src/state/apis/zapper/client.ts

```diff
@@ -0,0 +1,984 @@
+import { type ZodiosOptions, makeApi, Zodios } from '@zodios/core'
+import { z } from 'zod'
+
+const GasPricesResponse = z.object({
+  standard: z.object({}).partial(),
+  fast: z.object({}).partial(),
+  instant: z.object({}).partial(),
+  eip1559: z.boolean(),
+})
+
+export const schemas = {
```

Comment:
Will obviously need to abstract more things away, lots of repetition going on in this schema